test/platform/lists.relationshipTypes.xsd

```diff
@@ -0,0 +1,203 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These are the actual files that were *supposed* to have been included before. I had to find these online to make sure they are the right ones being included.